resources/js/store/modules/user/actions.js

```diff
@@ -146,8 +142,6 @@ export const actions = {
             }
         })
         .then(response => {
-            console.log('get_users_profile_map_data', response);
```

Comment:
Please put back these comments, the lowercase action name is very useful when debugging and checking incoming data locally

Reply:
Ok, I'm bringing them back.